### DISCUSSION TITLE
Shopware recipe locally compiled artefacts

### DISCUSSION THREAD

**Discussion by csteuer** [deployphp/deployer#3394] [Help needed]
Hi,

I'm trying to write a Shopware recipe based on the existing recipe for a docker-compose setup.

The existing Shopware recipe compiles the JS and CSS of storefront and administration locally: https://github.com/deployphp/deployer/blob/master/recipe/shopware.php#L163

However, I do not see where the locally compiled files are uploaded to the remote host.
The `deploy:update_code` clones the git repository, but does not upload any local files.

Am I missing something?

**Comment by marvinschroeder** [comment]
@csteuer I think the Shopware recipe was designed for deployments via CI/CD like GitHub Actions or GitLab CI/CD. For deployments started manually via vendor/bin/dep deploy from a local machine to the remote server, the recipe has to be customized in some parts.
